wiki/en/Software-Manual.md

```diff
@@ -77,16 +77,24 @@ open automatically for all Clients. See Settings to optionally turn on a sound a
 
 Opens a dialogue where you can select a server to connect to. If you are connected,
 pressing this button will end the session.
+You can also open the dialogue from File menu, Connection Setup.  This will leave you connected to any current session until you join another.
 
 <figure><img src="{% include img/en-screenshots/connection-setup-window.inc %}" style="border: 5px solid grey;" loading="lazy" alt="Screenshot of the server connection window"></figure>
 
-The Connection Setup window shows a list of available servers together with the number of occupants and the maximum number supported. Server operators register their servers on lists (mostly defined by genre, though some could be location-specific or for all genres). Use the List dropdown to select a genre, click on the server you want to join and press the Connect button to connect to it. Alternatively, double-click on the server name. Permanent servers (those that have been listed for longer than 24 hours) are shown in bold.
+The Connection Setup window shows a list of available Servers together with their "distance" (in terms of ping time), the number of occupants and the maximum number supported, and the Server's given location.
+Permanent Servers (those that have been listed for longer than 24 hours) are shown in bold.
 
-You can filter the list by server name or location. To list only occupied servers, enter a "#" character.
+Click on the Server you want to join and press the Connect button to connect to it. Alternatively, double-click on the server name.
 
-If you know the IP address or URL of a private server, you can connect to it using the Server Name/Address
-field. An optional port number can be added after the address using a colon as a separator, e.g,
-`jamulus.example.com:22124` (Note that IPv6 addresses must be entirely enclosed in square brackets). The field will also show a list of the most recently used server addresses.
+Server operators register their servers with Directories. These are either genre-specific or for any genre, or additional [custom entries](#custom-directories). Use the Directory dropdown to select a genre.
+You can filter the list by server name or location. To list only occupied Servers, enter a "#" character.
+If you choose "Show All Musicians" the current occupants will be shown.
+
+If you know the IP address or URL of a private Server, you can connect to it using the Server Name/Address field.
+An optional port number can be added after the address using a colon as a separator, e.g, `jamulus.example.com:22124`.
+(Note that IPv6 addresses must be entirely enclosed in square brackets.)
```

Comment:
Out of scope, but I'd add an example:


```suggestion
(Note that IPv6 addresses must be entirely enclosed in square brackets e.g.: `[::1]`.)
```

Reply:
Might want to include the port in that case (so would it be `[::1]:22124` or `[::1:22124]`?)

Reply:
It's outside the `[...]` in the regexp.